core/server/models/plugins/pagination.js

```diff
@@ -130,6 +128,11 @@ pagination = function pagination(bookshelf) {
         /**
          * ### Fetch page
          * A `fetch` extension to get a paginated set of items from a collection
+         *
+         * We trigger two queries:
+         * 1. count query to know how many pages left (important: we don't attach any group/order statements!)
+         * 2. the actualy fetch query with limit and page property
```

Comment:
😁   https://github.com/TryGhost/Ghost/issues/6103 - was written before bookshelf added a `fetchPage` method, but will add that in now